lib/scheduler.pm

```diff
@@ -25,6 +25,7 @@ use File::Basename;
 use testapi qw(get_var set_var);
 use main_common 'loadtest';
 use YAML::Tiny;
+use Data::Dumper;
```

Comment:
Let's either remove this import, or log parsed test data.
Other than that PR LGTM ;)

Reply:
I will print it, may help.

Reply:
I've checked and `Data::Dumper` is already used in os-autoinst, so no extra dependencies or risks that it's not deployed to the workers.